checkov/terraform/module_loading/loaders/registry_loader.py

```diff
@@ -31,7 +31,7 @@ def _is_matching_loader(self) -> bool:
         if os.path.exists(self.dest_dir):
             return True
 
-        self.module_version_url = os.path.join(self.REGISTRY_URL_PREFIX, self.module_source, "versions")
+        self.module_version_url = self.REGISTRY_URL_PREFIX + "/" + self.module_source + "/versions"
```

Comment:
```suggestion
        self.module_version_url = "/".join((self.REGISTRY_URL_PREFIX, self.module_source, "versions"))
```
a concatenation with "+" is most of the time discouraged, because it creates each time a new string, therefore using `.join()` is most of the time the better idea 🙂 
beware the double brackets are needed, because you need something to iterate through!

Reply:
I did know this I just forgot! Linked to immutability of python strings right. I will update today :)